src/abstractdataframe/reshape.jl

```diff
@@ -150,7 +150,7 @@ melt(df::AbstractDataFrame; variable_name::Symbol=:variable, value_name::Symbol=
 Unstacks a DataFrame; convert from a long to wide format
 
 ```julia
-unstack(df::AbstractDataFrame, rowkey, colkey, value)
+unstack(df::AbstractDataFrame, rowkeys, colkey, value)
```

Comment:
I would be useful to mention accepted types too. I guess you could have one line with `::Union{Symbol, Integer}`, and another one with `::AbstractVector{<:Union{Symbol, Integer}}`.

Reply:
fixed (although a bit messy)